### PR TITLE
Изменения в пакете mercury6 (внесены idovgalyov).

### PR DESCRIPTION
**Pull request by idovgalyov**
What is changed:
In all files (as possible), a double precision format for float numbers was added, in accordance with Fortran specifications.

List of important derivations:
*) Updating source files leaded to maximum differences in order of 10-11 symbol after point (sth about tens of meters in case of checking cartesian coords and major semiaxis)
*) Input data provided by user must have a double precision format (i.e. for each float number in "big.in" file that number must end with "d" exponent (and "e" exponent must be replaced by "d" everywhere). Approaching difference was in 10-11 symbol for big planets (<100m) and in 8-9 symbol (~1km) for NEA per one century of integrating time (checked on Apollo).
*) Crash-test was performed using Apophis (with initial data from NASA web-site). The test was performed on Everhart and Boulirsh-Stoere algorithms before and after modification. The difference between the distances of the close approach in 2029 raises up to order of kilometers. After a hundred of years it approaches a level of 2000-3000 km (including that between BS results). The difference of Everhart and BS results is a little smaller in contrast to original version.
*) Noticed: The program uses the Everhart method for adjusting all bodies to a start epoch. This must be taken into account, especially with the presence of close encounters between starting and initial epochs.
*) I did not touch formats of internal (not for user) input-output.

Something more: added new initial data for planets and updated "small.in.sample" file with new data for Apollo and Apophis.
I compared close encounters output with the tables of NASA.

Accordance with NASA for the next hundred years for Apolllo is satisfactory (for the latest time there will be a difference >1000km). The result is worse for Apophis, however, the important dates are complied.
In general, the list of encounters became more compatible with the data from NASA.

<2016-12-05>
Included description in ReadMe file (about main updates), and there is also the source for planets' initial data (http://ssd.jpl.nasa.gov)
Also reset some parameters in "params.in.sample"

### CODE REVIEW

File: big.in.sample [hunk 1]
```diff
@@ -2,41 +2,41 @@
...
- NEPTUNE    m=5.15138902046611451E-05 r=3.d0 d=1.76
-  1.53796971148509165E+01 -2.59193146099879641E+01  1.79258772950371181E-01
-  2.68067772490389322E-03  1.62824170038242295E-03 -9.51592254519715870E-05
-  0. 0. 0.
- PLUTO      m=7.39644970414201173E-09 r=3.d0 d=1.1
- -1.15095623952731607E+01 -2.70779438829451422E+01  6.22871533567077229E+00
-  2.97220056963797431E-03 -1.69820233395912967E-03 -6.76798264809371094E-04
-  0. 0. 0.
+ epoch (in days) = 2457724.5D0
+)----------------------------------------------------d or D is not matter--0d0 is possible too
```
Comment (smirik) [inline big.in.sample]: Please provide the source of the initial data for the planets.

File: close.in.sample [hunk 2]
```diff
@@ -7,7 +7,7 @@
  ce.out
 )---------------------------------------------------------------------
  express time in days or years = years
- express time relative to integration start time = yes
+ express time relative to integration start time = no
```
Comment (smirik) [inline close.in.sample]: Why no?

File: param.in.sample [hunk 3]
```diff
@@ -3,11 +3,11 @@
...
 )---------------------------------------------------------------------
- algorithm (MVS, BS, BS2, RADAU, HYBRID etc) = bs
- start time (days)= 2451000.5
- stop time (days) = 2501000.5
- output interval (days) = 365.25d0
- timestep (days) = 8
+ algorithm (MVS, BS, BS2, RADAU, HYBRID etc) = radau
+ start time (days)= 2454441.5d0
+ stop time (days) = 2501000.5d0
+ output interval (days) = 101.d0
```
Comment (smirik) [inline param.in.sample]: Why?

**Comment by smirik** [comment]
1. I would recommend to use english instead of russian.
2. Please update the README and/or manual file and describe the changes you've made.
3. Please provide the source of the data.

File: big.in.sample (hunk 1 above)
Comment (idovgalyov) [inline big.in.sample]: The initial data was received from
http://ssd.jpl.nasa.gov/
by using HORIZONS web-interface (it provides several positions on a user-given time interval, and I took one of them)

File: close.in.sample (hunk 2 above)
Comment (idovgalyov) [inline close.in.sample]: I used this value to make easier comparing with list of close approaches (e. g. for Apophis), where both Julian and Gregorian dates are given.
http://ssd.jpl.nasa.gov/sbdb.cgi?sstr=apophis;old=0;orb=0;cov=0;log=0;cad=1#cad
As described in mercury6 manual, that value and some others in ".in" files can be changed by user if he wants to specify the output format

File: param.in.sample (hunk 3 above)
Comment (idovgalyov) [inline param.in.sample]: As i said above, those values can be freely altered depending on what we want to get in output.

**Pull request merged**